config/pmd-test.xml

```diff
@@ -99,9 +97,7 @@
               or @Image='SuppressWithNearbyCommentFilterTest' or @Image='SuppressionCommentFilterTest'
               or @Image='SuppressionFilterTest' or @Image='SuppressionXpathFilterTest'
               or @Image='SuppressWithPlainTextCommentFilterTest' or @Image='XpathFilterTest']//PrimarySuffix[@Image='verify']
-            | //ClassOrInterfaceDeclaration[@Image='PackageObjectFactoryTest']//MethodDeclarator[@Image='testCreateModuleWithNonExistName']
-            | //ClassOrInterfaceDeclaration[@Image='XdocsPagesTest']//MethodDeclarator[@Image='testAllChecksPresentOnAvailableChecksPage']
-            "/>
```

Comment:
please restore this, `"/>` being alone on the next line, separate from the other suppressions.